config/config.go

```diff
@@ -148,6 +151,9 @@ func init() {
 	cfg.SetDefault("storage.elasticsearch.retry", 60)
 	cfg.SetDefault("storage.elasticsearch.bulk_maxdocs", 100)
 	cfg.SetDefault("storage.elasticsearch.bulk_maxdelay", 5)
+	cfg.SetDefault("storage.elasticsearch.index_entries_limit", -1)
+	cfg.SetDefault("storage.elasticsearch.index_age_limit", -1)
+	cfg.SetDefault("storage.elasticsearch.indices_to_keep", -1)
 	cfg.SetDefault("storage.orientdb.addr", "http://localhost:2480")
```

Comment:
nit: better to order these alphabetically:
- index_age_limit
- index_entries_limit
- indices_to_keep